packages/roosterjs-editor-dom/lib/utils/adjustNodeInsertPosition.ts

```diff
@@ -17,7 +30,76 @@ import { NodePosition, NodeType, PositionType } from 'roosterjs-editor-types';
  * @param position The original position to insert the node
  */
 export default function adjustNodeInsertPosition(
-    root: Node,
+    root: HTMLElement,
```

Comment:
I noticed you changed the `root `type from `Node `to `HTMLElement`. Was this related to the fix for the bug?

Reply:
Yes. Because API queryElements() requires HTMLElement. And here root actually must be an HTMLElement.